### DISCUSSION TITLE
Setup a default entry page per Role

### DISCUSSION THREAD

**Discussion by supermx1** [directus/directus#8751] [Feature Requests]
So under the Roles & Permissions page > New role 

We can set up a default page so that on sign-in, it automatically takes the user to that pre-defined page. 

UI: maybe when creating a Collection Navigation, we can check a box that says make this page default page.

**Comment by benhaynes** [comment]
We'll start with a global default page, and then can add a role override.

**Reply by supermx1** [reply]
We can do that. 

The global default page will be under the Project Settings page. For granular control, the default page for each role & permission will be under the Roles & Permissions page > New role page.

**Comment by br41nslug** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
